recipes/xxsds-sdsl-lite/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+import os
+from conans import ConanFile, tools
+
+
+class XXSDSSDSLLite(ConanFile):
+    name = "xxsds-sdsl-lite"
+    description = "SDSL - Succinct Data Structure Library"
+    homepage = "https://github.com/xxsds/sdsl-lite"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    topics = ("conan", "sdsl", "succint", "data-structures")
+    settings = "compiler"
+    exports = ["patches/*"]
+    provides = "sdsl-lite"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        source = self.conan_data["sources"][self.version]
+        tools.get(**source)
+        extracted_folder = "sdsl-lite-" + \
+            os.path.splitext(os.path.basename(source["url"]))[0]
+        os.rename(extracted_folder, self._source_subfolder)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version],
                  destination=self._source_subfolder, strip_root=True)
```
The `strip_root` parameter does some extra magic

https://docs.conan.io/en/latest/reference/tools.html#tools-get

Reply:
since my company uses conan 1.32.1, i'd like to keep it this way right now, and update the recipe in a follow-up PR

Reply:
Okay :+1: